### DISCUSSION TITLE
Astro templates

### DISCUSSION THREAD

**Discussion by louiss0** [nxtensions/nxtensions#347] [Q&A]
I'm wondering whether or not other Astro templates will ever be supported with this extension. Astro has many kinds of templates. 

I'm hoping that those could be used in some projects https://astro.new

**Comment by leosvelperez** [comment]
Hey @louiss0!

As mentioned in another discussion, please don't cross-post requests. The issue to support the templates is the right place to inquire about it and you've already done so.
